src/inc/msquic.h

```diff
@@ -703,6 +703,7 @@ void
 #define QUIC_PARAM_GLOBAL_GLOBAL_SETTINGS               0x01000006  // QUIC_GLOBAL_SETTINGS
 #ifdef QUIC_API_ENABLE_PREVIEW_FEATURES
 #define QUIC_PARAM_GLOBAL_VERSION_SETTINGS              0x01000007  // QUIC_VERSION_SETTINGS
+#define QUIC_PARAM_GLOBAL_LIBRARY_GIT_HASH              0x01000008  // char[64]
```

Comment:
Need to update C# and Rust, right?

Reply:
Rust does, but I'll need your help. I don't know how you'd read a string. Rust is also broken anyway, its param number is wrong.

Reply:
Don't worry about adding a function to actually query the value, just add/fix the global params.